app/assets/javascripts/components/story/StoryCopyIdClipboard.jsx

```diff
@@ -1,15 +1,10 @@
 import React from 'react';
-import Clipboard from 'react-clipboard.js';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
 
 const StoryCopyIdClipboard = ({ id }) => (
-  <Clipboard
-    data-clipboard-text={`#${id}`}
-    component="button"
-    className="story-id"
-    title={I18n.t('story.events.copy_id')}
-  >
-    #{id}
-  </Clipboard>
+  <CopyToClipboard text={I18n.t('story.events.copy_id')}>
```

Comment:
Same here:

```suggestion
  <CopyToClipboard text={`#${id}`}>
```

Reply:
Yes sure, you're right. Btw we need no translation because it is number and url. There is no necessity to translate number or urls.

Reply:
Done.